### PR TITLE
API1800 support for Volume attachment

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1800 support for Volume attachment]

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Enclosure
 - Enclosure Group
 - Uplink Sets
+- Storage Volume Attachment
```
Comment (AsisBagga) [inline CHANGELOG.md:14]: Please arrange the resources alphabetically

File: examples/storage_volume_attachment.go [hunk 2]
```diff
@@ -32,7 +33,12 @@ func main() {
 	for i := 0; i < len(attachment_list.Members); i++ {
 		fmt.Println(attachment_list.Members[i].URI)
 	}
+	id := filepath.Base(string(attachment_list.Members[0].URI))
 
+	// Get volume attachment by id
+	fmt.Println("\nGetting details of volume attachment with name: ", id)
+	volAttach_by_id, _ := ovc.GetStorageAttachmentById(id)
+	fmt.Println(volAttach_by_id)
```
Comment (AsisBagga) [inline examples/storage_volume_attachment.go:41]: Please add the if condition for err response.

Comment (nabhajit-ray) [reply examples/storage_volume_attachment.go:45]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (nabhajit-ray) [reply CHANGELOG.md:27]: Done

**Pull request merged**